src/StackHCI/custom/stackhci.ps1

```diff
@@ -115,6 +115,8 @@ $ConfiguringCloudManagementMessage = "Configuring Cloud Management agent."
 $ConfiguringCloudManagementClusterSvc = "Creating Cloud Management cluster resource."
 $StartingCloudManagementMessage = "Starting Cloud Management agent."
 
+$RemoteSupportConsentText = "`r`n`r`nBy approving this request, the Microsoft support organization or the Azure engineering team supporting this feature ('Microsoft Support Engineer') will be given direct access to your device for troubleshooting purposes and/or resolving the technical issue described in the Microsoft support case. `r`n`r`nDuring a remote support session, a Microsoft Support Engineer may need to collect logs.  By enabling remote support, you have agreed to a diagnostic logs collection by Microsoft Support Engineer to address a support case You also acknowledge and consent to the upload and retention of those logs in an Azure storage account managed and controlled by Microsoft.  These logs may be accessed by Microsoft in the context of a support case and to improve the health of Azure Stack HCI. `r`n`r`nThe data will be used only to troubleshoot failures that are subject to a support ticket, and will not be used for marketing, advertising, or any other commercial purposes without your consent.  The data may be retained for up to ninety (90) days and will be handled following our standard privacy practices (https://privacy.microsoft.com/en-US/). Any data previously collected with your consent will not be affected by the revocation of your permission."
```

Comment:
There are a few double-spaces ("  ") in this string, e.g. before "The data may be retained ...". That doesn't seem intentional. Are they supposed to be line-breaks?